khelo/lib/ui/flow/home/components/home_view_all_screen.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:data/api/match/match_model.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:khelo/components/app_page.dart';
+import 'package:khelo/components/match_detail_cell.dart';
+import 'package:khelo/ui/app_route.dart';
+
+class HomeViewAllScreen extends ConsumerWidget {
+  final String title;
+  final List<MatchModel> matches;
```

Comment:
instead of passing whole list, we should fetch matches as per passed status. also can you try pagination in this?